ui/src/routes/runs/$runId/index.tsx

```diff
@@ -28,19 +28,25 @@ export const Route = createFileRoute('/runs/$runId/')({
       const ortRun = await RunsService.getOrtRunById({
         runId: Number.parseInt(params.runId),
       });
-      organizationId = ortRun.organizationId;
-      productId = ortRun.productId;
-      repositoryId = ortRun.repositoryId;
-      index = ortRun.index;
+      organizationId = ortRun.organizationId.toString();
+      productId = ortRun.productId.toString();
+      repositoryId = ortRun.repositoryId.toString();
+      index = ortRun.index.toString();
```

Comment:
Sorry for a TS beginner question maybe, but how is this "typesafe" when longs are converted to strings here? Isn't this just faking typesafety, as instead of the implicit to-string-conversion in the interpolation of original line 42, you now just do it explicitly in these lines?

Reply:
Better type-safety is brought by feeding the `params` object to `redirect()`, which now recognizes the `to` as a valid route. I believe the earlier format was not type-safe in this manner.

Reply:
The PR was triggered by (supposedly) tightened type-checking of TanStack Router, which lead to [this error](https://github.com/eclipse-apoapsis/ort-server/actions/runs/12538694250/job/34975031666?pr=1711#step:8:49).